minvws/nl-kat-coordination
# Fix duplicate OOI references in result in origin

### Changes

Fix duplicate OOI references in result in origin

### Issue link

Closes #3508

### Demo

N/A

### QA notes

Two methods:
* Use VisualOctopoesStudio on a typical run to check there are no duplicate references in results
* Use `octopoes/tools/xtdb-cli.py query '{:query {:find [(pull ?var [*])] :where [[?var :type "Origin"]]}}' | jq -C  | less -R` and check there are no duplicates in a typical run

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
